Tests/PMKCoreLocation/CLGeocoderTests.swift

```diff
@@ -17,7 +17,7 @@ class CLGeocoderTests: XCTestCase {
         }
 
         let ex = expectation(description: "")
-        MockGeocoder().reverseGeocode(location: CLLocation()).done { x in
+        _ = MockGeocoder().reverseGeocode(location: CLLocation()).done { x in
```

Comment:
Let’s not do unnecessary things in PRs thanks. Tight PRs without other (even well intentioned) changes prevent review errors.

Reply:
this is just to get rid of the warning, but ok I can revert it